x-pack/plugins/fleet/public/components/linked_agent_count.tsx

```diff
@@ -21,8 +21,9 @@ export const LinkedAgentCount = memo<
     count: number;
     agentPolicyId: string;
     showAgentText?: boolean;
+    additionalKuery?: string;
   }
->(({ count, agentPolicyId, showAgentText, ...otherEuiLinkProps }) => {
+>(({ count, agentPolicyId, showAgentText, additionalKuery = '', ...otherEuiLinkProps }) => {
```

Comment:
nit: I'm wondering if the `unprivileged` query could be moved to this component, rather than passed as a prop. In case the unprivileged agent count will have to be reused somewhere else.

Reply:
👍 I moved it